solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -113,7 +115,7 @@ public class Http2SolrClient extends HttpSolrClientBase {
   private final HttpClient httpClient;
 
   private List<HttpListenerFactory> listenerFactory = new ArrayList<>();
-  private final AsyncTracker asyncTracker = new AsyncTracker();
+  protected AsyncTracker asyncTracker = new AsyncTracker();
```

Comment:
I missed why this was related; what's going on here @gerlowskija ?

Reply:
It's a visibility change to allow the NoCloseHttp2SolrClient to access this instance var, so that the "NoClose" client can use the same instance as the "normal" Http2SolrClient it derives from.  See [here](https://github.com/apache/solr/pull/2714/files#diff-db379c74efc9ff28acfd7571b07e16dbcdaa510783f2bee835b4a288927b5b3dR841)